cypress/support/cypress-support/src/main.js

```diff
@@ -189,6 +196,13 @@ export default function (module) {
             }
           });
         }
+        if (UTILS.getEnvVariable(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED) == true) {
+          cy.startOrStopInteractionsService(CONSTANTS.STOPPED).then((response) => {
+            if (response) {
+              Cypress.env(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED, false);
+            }
+          });
+        }
         // unsubscribing the list of topics
```

Comment:
is this befoerOperation  the equivalent of the glue code to start/stop fb interactions

Reply:
yes, you can use it in feature file or with before operation